README.md

```diff
@@ -190,6 +190,24 @@ The server may reject one or more events included in the cache. If this happens
 
 Global properties are evaluated and added when AddEvent() is called, so dynamic properties will not be evaluated when SendCachedEvents() is called.
 
+Analysis
+------------
+
+To run [analyses](https://keen.io/docs/api/#analyses) on your data, use the provided `KeenClient.Query` family of methods. For example:
+
+```
+var itemCount = keenClient.Query(QueryType.Count(), "target_collection", null);
+```
+
+An `async` version of this analysis could be run as follows:
+
+```
+var itemCount = await keenClient.Query(QueryType.Count(), "target_collection", null);
+```
+
+Additional qualifiers can be added to the analysis, such as the target property to use for analyses that require it. A timeframe or list of filters to use for the analysis can also be provided. If you'd like to get results in a grouped or time interval format, the `KeenClient.QueryGroup`, `KeenClient.QueryInterval`, and `KeenClient.QueryIntervalGroup` synchronous and asynchronous methods can be used. See the Grouped and Interval Query Results and Filters sections below for more detail.
```

Comment:
"A timeframe and/or list of filters" right? They aren't mutually exclusive, so the "or" alone seems a little strange.